mux.js

```diff
@@ -195,12 +195,15 @@ Multiplexer.prototype._onRemoteReplicate = function (keys) {
   })
 
   // Start replicating as requested.
-  this._replicateFeeds(filtered)
+  this._replicateFeeds(filtered, true, function () {
+    self.stream.emit('remote-feeds')
+  })
 }
 
 // Initializes new replication streams for feeds and joins their streams into
 // the main stream.
-Multiplexer.prototype._replicateFeeds = function(keys) {
+Multiplexer.prototype._replicateFeeds = function(keys, terminateIfNoFeeds, cb) {
```

Comment:
terminateIfNoFeeds doesn't do anything in this version -- should it?

Reply:
It's part of a fix made in 1c038d85b21cc4f4368fc4d813837a683c945773. I think it's ok (for now) to just let it be.